src/main/java/blockchain/Blockchain.java

```diff
@@ -1,16 +1,31 @@
 package blockchain;
 
 import java.nio.charset.StandardCharsets;
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
 import java.util.ArrayList;
 
+import com.google.common.hash.Hashing;
+
+//import org.apache.commons.codec.digest.DigestUtils.sha256Hex(stringText);
+
+
 public class Blockchain {
 	
-	Block current_block = new Block();
+	Block  current_block;
 	ArrayList<Block> blockchain = new ArrayList<>();
 	
-	public static void main(String[] args) {
+	public void main(String[] args) {
```

Comment:
If you need this setup only for the test you can move this is in a setUp function in JUnit.